src/test/java/uk/gov/pay/directdebit/events/api/DirectDebitEventsPaginationTest.java

```diff
@@ -0,0 +1,93 @@
+package uk.gov.pay.directdebit.events.api;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.mockito.stubbing.Answer;
+import uk.gov.pay.directdebit.payments.links.PaginationLink;
+import uk.gov.pay.directdebit.payments.params.DirectDebitEventSearchParams;
+
+import javax.ws.rs.core.UriBuilder;
+import javax.ws.rs.core.UriInfo;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class DirectDebitEventsPaginationTest {
+
+    
+    @Mock
+    private UriInfo mockUriInfo;
+
+    @Before
+    public void setup() {
+        when(mockUriInfo.getBaseUriBuilder()).thenAnswer((Answer<UriBuilder>) invocation -> UriBuilder.fromUri("http://test.com"));
```

Comment:
The domain name `test.com` is a real, registered domain name. Please use something like `whateveryoulike.test` — `.test` domains are guaranteed never to be registered (see RFC 2606)